geth/peers/peerpool_test.go

```diff
@@ -46,7 +47,7 @@ func (s *PeerPoolSimulationSuite) SetupTest() {
 	s.Require().NoError(s.bootnode.Start())
 	bootnodeV5 := discv5.NewNode(s.bootnode.DiscV5.Self().ID, net.ParseIP("127.0.0.1"), uint16(port), uint16(port))
 
-	s.peers = make([]*p2p.Server, 2)
+	s.peers = make([]*p2p.Server, 3)
```

Comment:
Would appreciate a constant for 3 describing why this size has been chosen?

Reply:
Ah, yes, that's why I always have troubles with those test suite packages: setup is too far away from the tests. I still prefer to have good old Go `TestFoo(t)` functions only using asserts in there. But that's a different topic. 😅